shell_scripts/init_cygwin_fixes.sh

```diff
@@ -29,3 +29,15 @@ then
     tar xvf /tmp/tar-1.32-2.tar.xz -C /
   fi
 fi
+
+###################### Fix binutils in 64 bit cygwin #####################
+
+# Cygwin updated to mingw64-x86_64-binutils is broken on Cygwin 64
+
+if [[ "$OSTYPE" == cygwin ]]
+then
+  if [ "`uname -m`" = "x86_64" ]; then
+    wget http://mirrors.kernel.org/sourceware/cygwin/x86_64/release/mingw64-x86_64-binutils/mingw64-x86_64-binutils-2.35.1-1.tar.xz -O /tmp/mingw64-x86_64-binutils-2.35.1-1.tar.xz
```

Comment:
As per zulip, I think you should be resilient to the file not being there.

Reply:
As is the script will stop with an error if the file is not there. I think this is what we want. The main script has `set -o errexit` (in a sourced init script).